.circleci/config.yml

```
@@ -604,12 +660,41 @@ jobs:
       - run:
           name: Publish CocoaPods Podspec
           command: pod trunk push
+      - run:
+          name: Inject new Podspec version to CDN cache
+          description: |
+            Force adding new version to the cache to avoid long running clone operation
+            The following command would take the latest CocoaPods version from cache.
```

```suggestion
            The following command takes the latest CocoaPods version from cache.
```

I don't understand what this means. Do you mean that it inserts the latest SDK version into the CocoaPods cache?

---

Right. Also the selected sentence mean that `scripts/validate-integrations/validate-integrations.sh` would depend on this changes.